opt/outliner/InstructionSequenceOutliner.cpp

```diff
@@ -1620,6 +1621,7 @@ class MethodNameGenerator {
   std::unordered_map<StableHash, size_t> m_unique_method_ids;
   size_t m_max_unique_method_id{0};
   size_t m_iteration;
+  std::atomic_int short_id_counter{0};
```

Comment:
Make that a `size_t`. `atomic` implies that there is some concurrent access going on, which there isn't. (As you might have noticed, we tend to use `size_t` to counted entities.)

Reply:
Updated!